kaggle_environments/main.py

```diff
@@ -230,21 +249,25 @@ def http_request(request):
             del params[key]
 
     body = request.get_json(silent=True, force=True) or {}
-    req = parse_args({**params, **body})
-    resp = action_handler(req)
+    args = {**params, **body}
+    if "render" in args:
+        # Manually deserialize render argument
+        # We should eventually refactor this to use the same deserializer as the cmd line arg parser
```

Comment:
Is it hard to add this in to the proper place for this PR?

Reply:
I wasn't able to find a good way to do it that didn't lead to a refactor that was out of scope unfortunately -- we likely will want to parse in two phases going forward, a cmd line parser that parses into a `Dict[str, str]` and then an arg parser that converts a `Dict[str, str]` into a nicely typed and deserialized args dictionary. Currently deserialization happens in one step that also parses the cmd line args so I couldn't see a good way to reuse that functionality for the http-server, but if you have a clean path forward here I'm definitely interested.